shoop/admin/templates/shoop/admin/orders/detail.jinja

```diff
@@ -126,6 +126,32 @@
                             </div>
                         </div>
                     {% endcall %}
+
+                    {% set payments=order.payments.all() %}
+                    {% if payments %}
+                        {% call content_block(_("Payments"), "fa-dollar") %}
+                            <table class="table table-striped table-bordered">
```

Comment:
This table is a fair bit simpler than the order content one (4 columns, most of which will have short contents).